app/components/HowCanIHelp/HelpMethods.tsx

```diff
@@ -112,8 +129,14 @@ const HelpMethods = ({
   <div className="help-footer">
     <h2 className="teal-500 padding-bottom-56">{footerTitle || (current && titles[current])}</h2>
     <div className="flexbox padding-bottom-80">
-      <KnowledgeCard current={current} />
-      <CommunityCard current={current} />
+      <KnowledgeCard
+        current={current}
+        className={`col-6 ${current === 'grassroots' || current === 'volunteer' || current === 'donate' ? 'order-2' : ''}`}
```

Comment:
an alternative way would be `${['grassroots', 'volunteer', 'donate'].includes(current) ? 'order-2' : ''}`